downstream/modules/builder/proc-customize-ee-image.adoc

```diff
@@ -62,11 +66,29 @@ localhost/new-ee    latest  f5509587efbb  3 minutes ago  769 MB
 ----
 ====
 . Verify your newly-created {ExecEnvShort} image via Ansible Navigator
+. Tag the image for use in your {HubName}:
++
+-----
+$ podman tag registry.redhat.io/_[username]_/_new-ee_ [automation-hub-IP-address]/_[username]_/_new-ee_
+-----
++
+. Log in to your {HubName} using Podman:
++
+[NOTE]
+=====
+You must have `admin` or appropriate container repository permissions for {HubName} to push a container. See _Managing containers in private automation hub_ for more information.
```

Comment:
perhaps a link to the "managing containers" doc might be useful here

Reply:
@TarikFD I linked to the docs landing page since direct docs linking requires specifying a version # and I'd rather future-proof this against needing to be updated in the Spring.

Reply:
makes sense